app/src/main/java/com/example/mymovieinfo/dto/Specimen.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.mymovieinfo.dto
+
+data class Specimen(var movieTitle: String = "",
+               var movieRank: Int = 0,
+               var movieCountry : String = "",
+               var movieBoxOfficeGross: String = "",
+               var movieOpeningWeekendGross : String = "",
+               var movieDistributor : String = "")
+{
+    override fun toString(): String {
+        return movieTitle
+    }
+}
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.